src/main.js

```diff
@@ -8,6 +8,7 @@ import Master from '@/Role/Master'
 import Login from '@/Login'
 
 import BootstrapVue from 'bootstrap-vue'
+import 'bootstrap/dist/css/bootstrap.css'
```

Comment:
What's about these in Role/Admin.vue?
@import "../../custom-bootstrap.scss";
@import "../../node_modules/bootstrap/scss/bootstrap.scss";

Reply:
good point. It did not load css for b-form-select and show different select box in firefox and chrome browser